apps-rendering/src/components/layout/standard.stories.tsx

```diff
@@ -0,0 +1,77 @@
+// ----- Imports ----- //
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { breakpoints } from '@guardian/source-foundations';
+import { partition } from '@guardian/types';
+import { withKnobs } from '@storybook/addon-knobs';
```

Comment:
I'd recommend against adding new usages of this library; it was deprecated in favour of `addon-controls`.

You could replace one with the other, but I'm not sure any of these stories actually use controls, so perhaps it could be dropped from here entirely?